### ISSUE TITLE
0.18 package on PyPi ships 0.16

### ISSUE THREAD

**Issue by daenney** [ua-parser/uap-python#186]
I just tried to install ua-parser 0.18.0 from PyPi, but it results in 0.16 getting installed:

```
~/D/c/apub ❯❯❯ curl -LO https://files.pythonhosted.org/packages/72/c3/5e715a16a9f8117a9ed92f5112911e07d40cfdf0f81cd4139f0c6307c57c/ua-parser-0.18.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 40659  100 40659    0     0   169k      0 --:--:-- --:--:-- --:--:--  170k

~/D/c/apub ❯❯❯ tar zxvf ua-parser-0.18.0.tar.gz                                                                  (base) 
ua-parser-0.18.0/
ua-parser-0.18.0/MANIFEST.in
ua-parser-0.18.0/PKG-INFO
ua-parser-0.18.0/README.rst
ua-parser-0.18.0/setup.cfg
ua-parser-0.18.0/setup.py
ua-parser-0.18.0/ua_parser/
ua-parser-0.18.0/ua_parser/LICENSE
ua-parser-0.18.0/ua_parser/__init__.py
ua-parser-0.18.0/ua_parser/_regexes.py
ua-parser-0.18.0/ua_parser/user_agent_parser.py
ua-parser-0.18.0/ua_parser/user_agent_parser_test.py
ua-parser-0.18.0/ua_parser.egg-info/
ua-parser-0.18.0/ua_parser.egg-info/PKG-INFO
ua-parser-0.18.0/ua_parser.egg-info/SOURCES.txt
ua-parser-0.18.0/ua_parser.egg-info/dependency_links.txt
ua-parser-0.18.0/ua_parser.egg-info/not-zip-safe
ua-parser-0.18.0/ua_parser.egg-info/top_level.txt
~/D/c/apub ❯❯❯ cat ua-parser-0.18.0/ua_parser/__init__.py                                                        (base) 
VERSION = (0, 16, 1)
```

It seems the wrong package got uploaded.

The same is true for the wheel:

```
~/D/c/apub ❯❯❯ curl -LO https://files.pythonhosted.org/packages/db/d5/3cf1bdc3401dda1c45894deac836aaab571c26858f83e1b6d6a7c249d444/ua_parser-0.18.0-py2.py3-none-any.whl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 38314  100 38314    0     0  1377k      0 --:--:-- --:--:-- --:--:-- 1385k

~/D/c/apub ❯❯❯ /usr/bin/unzip ua_parser-0.18.0-py2.py3-none-any.whl                                              (base) 
Archive:  ua_parser-0.18.0-py2.py3-none-any.whl
  inflating: ua_parser/__init__.py   
  inflating: ua_parser/_regexes.py   
  inflating: ua_parser/user_agent_parser.py  
  inflating: ua_parser/user_agent_parser_test.py  
  inflating: ua_parser-0.18.0.dist-info/METADATA  
  inflating: ua_parser-0.18.0.dist-info/WHEEL  
  inflating: ua_parser-0.18.0.dist-info/top_level.txt  
  inflating: ua_parser-0.18.0.dist-info/RECORD  

~/D/c/apub ❯❯❯ cat ua_parser/__init__.py                                                                         (base) 
VERSION = (0, 16, 1)
```

This results in `from ua_parser import parse` not working:

```
>>> from ua_parser import parse
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'parse' from 'ua_parser'
```

**Comment by masklinn** [comment]
> I just tried to install ua-parser 0.18.0 from PyPi, but it results in 0.16 getting installed:

If you look at [the last commit of the 0.x branch](https://github.com/ua-parser/uap-python/commit/ae6398d219ba4b717e50f64be94f0c8e9768e16b) I didn't remember to update the `VERSION` stricture when I updated to 0.18. I did update the package metadata though, and that's what you'll find e.g. in `ua_parser-0.18.0.dist-info/METADATA` of the wheel (and in the setup.py of the tgz).

> This results in from ua_parser import parse not working:

That won't work either way, this is the not-yet-released 1.0 API. For the readme of 0.x you want https://github.com/ua-parser/uap-python/tree/0.x.

I guess I should add a warning at the top of the readme.
